packages/phrases/src/locales/en/translation/admin-console/sign-in-exp/index.ts

```diff
@@ -1,4 +1,4 @@
-import others from './others.js';
```

Comment:
Seems the `others.ts` is not removed from `/en` folder?

Reply:
a mistake